CMakeLists.txt

```diff
@@ -2776,6 +2776,9 @@ if(QML)
     src/qml/qmlplayerproxy.cpp
     src/qml/qmlvisibleeffectsmodel.cpp
     src/qml/qmlwaveformoverview.cpp
+    src/qml/mixxcomponentlifecycle.cpp
```

Comment:
I'm not sure this here is usefule. `MixxxComponentLifecycle` has no implementation since it's an interface but compilation fails without it.

Reply:
yeah, I think thats due to the MOC. Just keep it.